src/test/java/org/helpberkeley/memberdata/v300/ControlBlockBuilder.java

```diff
@@ -176,6 +177,12 @@ public ControlBlockBuilder withOpsManager(String opsManager) {
         return this;
     }
 
+    public ControlBlockBuilder withOpsManager(String opsManager, String macro) {
+        this.opsManager = opsManager;
+        this.builderOpsManagerMacro = macro;
+        return this;
+    }
+
```

Comment:
This is an optional suggestion...  I avoid unnecessarily overloading methods. It makes easier moving forward when new requirements arise and you need to pass in more optional data.  With overloaded methods this can lead to problems:
```
    void something(String arg1);
    void something(String arg1, String arg2);  

    // Now we add a need to do something with an arg3, which can work
   //  both with and without arg2.  So we would have to do:

   void something(String arg1);
   void something(String arg1, String arg2);  
   void something(String arg1, String arg2, String arg3);
   // But we can't express the following because it looks the the same signature
   // as something(arg1, arg2)
   void something(String arg1, String arg3);

This also touches on another reason that I would add withOpsManagerMacro() as a separate setter...when you have multiple parameters of the same type, it is easy for the caller to make a parameter ordering mistake when invoking the method.